packages/dialtone-vue2/components/tooltip/tooltip.vue

```diff
@@ -522,19 +511,25 @@ export default {
         this.anchor.removeEventListener(listener, (event) => this.onLeaveAnchor(event));
       });
     },
-
-    onTouchStart () {
-      this.isTouchDevice = true;
-    },
   },
 };
 </script>
 
 <style lang="less">
+@import 'tippy.js/dist/svg-arrow.css';
+
 .tippy-box[data-popper-reference-hidden] {
   .d-tooltip {
     visibility: hidden;
     pointer-events: none;
   }
 }
+
+.tippy-box > .tippy-svg-arrow {
+  fill: var(--dt-color-surface-contrast);
+}
+
+.tippy-box[data-animation='fade'][data-state='hidden'] {
+  opacity: 0;
+}
```

Comment:
Could we move this to dialpad css?, since this is not an scoped style, we will be importing `@import 'tippy.js/dist/svg-arrow.css';` globally when this component is being initialized.

Reply:
The problem is tippy.js is not a dependency of dialtone-css, so it won't be able to import it. Scoping it also won't work because the tooltip renders outside of the component at the document body. I think it's probably okay just to leave global 🤷